src/Offer/DefaultOfferEditingService.php

```diff
@@ -252,7 +253,7 @@ public function updatePriceInfo(string $id, PriceInfo $priceInfo)
         $this->guardId($id);
 
         return $this->commandBus->dispatch(
```

Comment:
I don't think this dispatch is needed? Even the method `updatePriceInfo` can be removed from the `OfferEditingServiceInterface` interface.